src/main/kotlin/org/jetbrains/intellij/tasks/VerifyPluginTask.kt

```diff
@@ -60,8 +60,13 @@ open class VerifyPluginTask @Inject constructor(
     fun verifyPlugin() {
         val creationResult = pluginDir.get().let { IdePluginManager.createManager().createPlugin(it.asFile.toPath()) }
         when (creationResult) {
-            is PluginCreationSuccess -> creationResult.warnings.forEach {
-                warn(context, it.message)
+            is PluginCreationSuccess -> {
+                creationResult.warnings.forEach {
+                    warn(context, it.message)
+                }
+                creationResult.unacceptableWarnings.forEach {
```

Comment:
Technically these must not necessarily be "errors" in every case. E.g. a plugin that will never be uploaded to JB MP might not care about these problems. There must be way to suppress/configure this for such cases.

Reply:
Ok, will add. Should it be something like `ignoreUnacceptableWarnings` or `ignoreMarketplaceErrors`? The default value should be false, right?
Also, see that using the default value in name, description, etc. is considered an error. Should it be so? Or should we make it an unacceptable warning, so the vendors can suppress it also in the same way?

Reply:
IMHO `ignoreUnacceptableWarnings`, default `false`
IMHO we can keep error level for all "default value" violations, even if they use plugin internally only, it makes sense to avoid these